test/memkind_memtier_test.cpp

```diff
@@ -2,6 +2,7 @@
 /* Copyright (C) 2021 Intel Corporation. */
 
 #include <memkind_memtier.h>
+#include <thread>
```

Comment:
nit: Add an extra blank line to split memkind header from standard library header.